libs/components/forms/src/lib/modules/radio/radio.component.html

```diff
@@ -30,5 +30,13 @@
   >
     <sky-icon *ngIf="icon" [fixedWidth]="true" [icon]="icon" />
   </span>
-  <ng-content select="sky-radio-label" />
+  <ng-container *ngIf="labelText; else labelElement">
+    <sky-radio-label-text-label
```

Comment:
what is the reasoning for creating this whole new component? all it seems to do is add a class, so i thjink the logic can easily be moved here and the classes added to a span

Reply:
removed